client/lib/signup/test/signup/config/flows.js

```diff
@@ -25,8 +25,6 @@ var flows = {
 };
 
 module.exports = {
-	currentFlowName: 'simple_flow',
```

Comment:
`currentFlowName` is the flow to which users visiting `/start` are redirected. The tests below look exhaustive, and the PR in general looks good. I'll give it some more thought.